### PR TITLE
Add support for 2.12.14

### PR DESCRIPTION
**Pull request by dos65**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -225,6 +226,7 @@ lazy val V = new {
     Seq(
       scala213,
       scala212,
+      "2.12.14",
```
Comment (tgodzik) [inline build.sbt:229]: You should change scala212 to be `2.12.14`, we try to keep on the latest version in Metals, and add `2.12.13` here. You will most likely need to set ammonite212Version to `2.12.14` if it's not yet published.

Also, please update `bin/test-release.sh`

Comment (dos65) [reply build.sbt:229]: scalafix isn't published for 2.12.14 yet and fails with:
```
[error] error: The ExplicitResultTypes rule only supports the exact Scala versions 
'List(2.12.13, 2.12.11, 2.12.12)' for this binary version.
```

Comment (tgodzik) [reply build.sbt:229]: ach, ok, we can update it later.

Comment (dos65) [reply build.sbt:229]: and the same for ammonite

**Review by tgodzik — APPROVED** [review]
LGTM!

File: build.sbt [hunk 2]
```diff
@@ -134,6 +134,7 @@ inThisBuild(
     ),
     testFrameworks := List(),
     resolvers += Resolver.sonatypeRepo("public"),
+    resolvers += Resolver.sonatypeRepo("snapshot"),
```
Comment (ckipp01) [inline build.sbt:137]: I know this is already merged, but does the snapshot resolver need to stay in here?

Comment (tgodzik) [reply build.sbt:137]: It's useful for testing out snapshot releases. We can remove it for sure, but it shouldn't matter too much.

**Pull request merged**